### PR TITLE
Add sass-lint, so our Sass is linty fresh

### LINKED ISSUE #93: Use sass-lint
More thanks to @pdehaan here: https://github.com/mozilla/pulse/pull/91#discussion_r100652857

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #93

### CODE REVIEW

**Comment by chuckharmston** [comment]
Tempted to not review this on the basis of the pun alone.

File: package.json [hunk 1]
```diff
@@ -52,8 +54,10 @@
     "package": "npm run build && mv dist/pulse.xpi addon.xpi",
     "deploy": "deploy-txp",
     "watch": "NODE_ENV=dev webpack --watch",
-    "prettier": "prettier --single-quote --write src/**/*.{js,jsx}",
-    "lint": "eslint . --ext .js --ext .jsx",
+    "prettier": "prettier --single-quote --write 'src/**/*.{js,jsx}'",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint . --ext .js --ext .jsx",
+    "lint:sass": "sass-lint -q -v",
```
Comment (chuckharmston) [inline package.json]: Can you please add `'src/**/*.scss'` to this to ensure that we're not linting module files?

Comment (chuckharmston) [inline package.json:60]: Can you please add `'src/**/*.scss'` to this to ensure that we're not linting module files?

**Review by chuckharmston — CHANGES_REQUESTED** [review]
Other than one comment, looks great and passes. Feel free to self-merge once addressed.

File: package.json (hunk 1 above)
Comment (pdehaan) [reply package.json:60]: Good catch, @chuckharmston! I forgot to include the .sass-lint.yml file.
See https://github.com/mozilla/pulse/pull/102/files#diff-c0745258b9513d87ab38c11820efcdfbR5

**Review by chuckharmston — APPROVED** [review]

**Pull request merged**